clients/new_gsql_version.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# called by make_aio_gsql_client.sh
+
+SRC=$1   # path of the gsql client code: com/tigergraph
+BRN=$2   # source branch: should be sth like tg_2.4.0_dev
+VSTR=$3  # version_string: should be sth like v2_4_0
+
+#Step 0:  add this version in AIO driver
+cat <<EOT >> com/tigergraph/client/Driver.java
+        try {
+            System.out.println("trying $VSTR");
+            com.tigergraph.$VSTR.client.Driver.main(args);
+        } catch (SecurityException e) {
+            ;
+        }
+EOT
+
+#Step 1: clean up and then create the target dir
+cd com/tigergraph
+rm -rf $VSTR;  mkdir $VSTR
+
+#Step 2: switch the source to correct branch (release). Get client commit
+cd $SRC; 
+git checkout $BRN; git pull; 
```

Comment:
This might not be working every time. Suggest to use:
git fetch --all -p
git checkout $BRN
git reset --hard origin/$BRN
git pull